### ISSUE TITLE
Typography/Link Styles - Meaningful link purpose

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#6]
**Requirements**:
- Link text should contain specific information related to the purpose of the link.

**Example**: Link text such as ‘Read more’ must be avoided, and text such as 'Read more about the conditions of transit workers' could be used
instead, the extra text can be hidden from visual display so long as it is present for screen reader users.

**Why is this required?** Screen reader users may tab through links to skim web pages, or may use a command to list all links on a page. In these cases the links will be read out of context and any ambiguous links (e.g. click here, read more) will be meaningless. 

**WCAG Reference**: This corresponds to WCAG 2.4.4 http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html

**Comment by papafed** [comment]
We fall foul of this _all the time_ - price plan grids, device listings, etc. Especially when built by sitebuilders.
Therefore as well as making sure that as devs we implement accessible link descriptions, when building the templates we need to provide a quick and easy enabler to allow SBs to automatically expand 'buy now' and 'click here' links to drag in titles or something.
We also need to push back every single time copywriters and legal insist that 'click here' is adequate text for a link.

**Comment by VinceButcher** [comment]
It is acceptable to have click here as long as we label the link, button correctly with relevant copy, so copywriters should be providing the link copy and the accessibility copy as well

**Comment by jinky32** [comment]
This was something that we / Frysla came up against with the Cookie banner.  Because the anchor text couldn't be changed we suggested they implement http://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/C7

**Comment by papafed** [comment]
Yep, that's what we ended up doing, but really they should have just let us turn the whole sentence into a link.
